### PR TITLE
create webp format

### PR DESCRIPTION
**Pull request by excellencetechnologies**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -1 +1,9 @@
-config = {'APP_ID': 'com.tethr', "DbUrl":"mongodb://127.0.0.1/magento_mobile", "URL": "http://144.76.34.244:8080/magento/mobile/web/index.php/excellence/mobile/api/v1", "CATEGORY_EXPIRESAT": "216000", "PRODUCT_EXPIRESAT": "216000", "ACCOUNT_EXPIRESAT":"21600", "IMAGE_URL" :"http://144.76.34.244:8080/shekhar_works/Eexpress_magento/public/", "IMAGE_MINIFY_URL" :"http://144.76.34.244:8080/shekhar_works/Eexpress_magento/public/minify/", "DEFAULT_IMAGE_URL":"http://144.76.34.244:8080/shekhar_works/Eexpress_magento/public/default/default.jpg"}
\ No newline at end of file
+config = {'APP_ID': 'com.tethr',
+    "DbUrl": "mongodb://127.0.0.1/magento_mobile",
+    "URL": "http://144.76.34.244:8080/magento/mobile/web/index.php/excellence/mobile/api/v1",
+    "CATEGORY_EXPIRESAT": "216000",
+    "PRODUCT_EXPIRESAT": "216000",
+    "ACCOUNT_EXPIRESAT": "21600",
+    "IMAGE_URL": "http://d3r38l4d4yyrur.cloudfront.net/",
```
Comment (upexcel) [inline config/index.js]: rename to CDN_URL

File: config/index.js [hunk 2]
```diff
@@ -1 +1,9 @@
-config = {'APP_ID': 'com.tethr', "DbUrl":"mongodb://127.0.0.1/magento_mobile", "URL": "http://144.76.34.244:8080/magento/mobile/web/index.php/excellence/mobile/api/v1", "CATEGORY_EXPIRESAT": "216000", "PRODUCT_EXPIRESAT": "216000", "ACCOUNT_EXPIRESAT":"21600", "IMAGE_URL" :"http://144.76.34.244:8080/shekhar_works/Eexpress_magento/public/", "IMAGE_MINIFY_URL" :"http://144.76.34.244:8080/shekhar_works/Eexpress_magento/public/minify/", "DEFAULT_IMAGE_URL":"http://144.76.34.244:8080/shekhar_works/Eexpress_magento/public/default/default.jpg"}
\ No newline at end of file
+config = {'APP_ID': 'com.tethr',
+    "DbUrl": "mongodb://127.0.0.1/magento_mobile",
```
Comment (upexcel) [inline config/index.js]: rename to DB_URL

File: config/index.js [hunk 3]
```diff
@@ -1 +1,9 @@
...
+    "URL": "http://144.76.34.244:8080/magento/mobile/web/index.php/excellence/mobile/api/v1",
+    "CATEGORY_EXPIRESAT": "216000",
+    "PRODUCT_EXPIRESAT": "216000",
+    "ACCOUNT_EXPIRESAT": "21600",
+    "IMAGE_URL": "http://d3r38l4d4yyrur.cloudfront.net/",
+    "DEFAULT_IMAGE_URL": "http://d3r38l4d4yyrur.cloudfront.net//default/default.webp"
```
Comment (upexcel) [inline config/index.js]: this should be png image and remove two slashes

File: service/image.js [hunk 4]
```diff
@@ -26,6 +26,9 @@ resize = function (url, APP_ID, mobile_width, callback) {
         var url_last_index_length = url.lastIndexOf('/');
         var image_name = url.substring(url_last_index_length + 1);
         var filename = image_stored_url.substring(0, image_stored_url.lastIndexOf("/"));
+        var image_name_without_extension = image_name.substr(0, image_name.lastIndexOf('.'));
+        var image_new_name = '/' + image_name_without_extension + '.webp';
```
Comment (upexcel) [inline service/image.js]: rename to image_webp

File: service/image.js [hunk 5]
```diff
@@ -26,6 +26,9 @@ resize = function (url, APP_ID, mobile_width, callback) {
...
+        var image_new_name_ = '/' + image_name_without_extension + '.png';
```
Comment (upexcel) [inline service/image.js]: rename to image_png

File: service/image.js [hunk 6]
```diff
@@ -55,40 +62,42 @@ resize = function (url, APP_ID, mobile_width, callback) {
...
+minify = function (url, APP_ID, callback) {
+    if (url && APP_ID) {
+        var image_url = URL_.parse(url).path;
+        var image_fetch_url = image_url.replace("/shekhar_works/Eexpress_magento/public/", "");
+        var filename = image_fetch_url.substring(0, image_fetch_url.lastIndexOf("/"));
+        var url_last_index_length = url.lastIndexOf('/');
+        var image_name = url.substring(url_last_index_length + 1);
+        var image_name_without_extension = image_name.substr(0, image_name.lastIndexOf('.'));
+        var image_new_name = '/' + image_name_without_extension + '.jpg';
+        var image_minified_name = filename.replace("comtethr/300", "comtethr/300/minify");
```
Comment (upexcel) [inline service/image.js]: cannot hard code data

File: service/image.js [hunk 7]
```diff
@@ -55,40 +62,42 @@ resize = function (url, APP_ID, mobile_width, callback) {
...
-         var image_fetch_url = image_url.replace("/shekhar_works/Eexpress_magento/public/", "");
-         var filename = image_fetch_url.substring(0, image_fetch_url.lastIndexOf("/"));
-         var url_last_index_length = url.lastIndexOf('/');
-         var image_name = url.substring(url_last_index_length + 1);
-         if (fileExists('public/minify/' + filename + '/' + image_name) == false) {
-                  imagemin(["public/original_image/" + image_name], 'public/minify/' + filename, {
+minify = function (url, APP_ID, callback) {
+    if (url && APP_ID) {
+        var image_url = URL_.parse(url).path;
+        var image_fetch_url = image_url.replace("/shekhar_works/Eexpress_magento/public/", "");
```
Comment (upexcel) [inline service/image.js]: cannot hard code

File: service/image.js [hunk 8]
```diff
@@ -55,40 +62,42 @@ resize = function (url, APP_ID, mobile_width, callback) {
...
+        var filename = image_fetch_url.substring(0, image_fetch_url.lastIndexOf("/"));
+        var url_last_index_length = url.lastIndexOf('/');
+        var image_name = url.substring(url_last_index_length + 1);
+        var image_name_without_extension = image_name.substr(0, image_name.lastIndexOf('.'));
+        var image_new_name = '/' + image_name_without_extension + '.jpg';
```
Comment (upexcel) [inline service/image.js]: why are we using jpg

File: config/index.js (hunk 3 above)
Comment (excellencetechnologies) [inline config/index.js]: remove slash

File: config/index.js (hunk 1 above)
Comment (excellencetechnologies) [inline config/index.js]: changed name

File: config/index.js (hunk 2 above)
Comment (excellencetechnologies) [inline config/index.js]: changed name to DB_URL

File: service/image.js (hunk 4 above)
Comment (excellencetechnologies) [inline service/image.js]: name changed to image_webp

File: service/image.js (hunk 5 above)
Comment (excellencetechnologies) [inline service/image.js]: changed ame to image_png

**Pull request merged**